contracts/SpokePool.sol

```diff
@@ -180,4 +261,34 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
     function chainId() public view returns (uint256) {
         return block.chainid;
     }
+
+    /**************************************
+     *         INTERNAL FUNCTIONS         *
+     **************************************/
+
+    function _computeAmountPreFees(uint256 amount, uint256 feesPct) private pure returns (uint256) {
+        return (1e18 * amount) / (1e18 - feesPct);
+    }
+
+    function _computeAmountPostFees(uint256 amount, uint256 feesPct) private pure returns (uint256) {
+        return (amount * feesPct) / 1e18;
```

Comment:
nit: should this also be `(1e18 - feesPct)` rather than just `feesPct`?

Reply:
shoot yeah, the test case where the fees are equally set to 0.25 and 0.25 is confusing because 1e18 - totalFeesPct == totalFeesPct. I changed the test values to 0.1 and 0.2 and changed this solidity code to `1e18 - feesPct`